### PR TITLE
Skip UI tests on aarch64_mac

### PR DESCRIPTION
**Pull request by gdams**
This was originally implemented in https://github.com/adoptium/aqa-tests/pull/2886 but appears to have been lost in the recent refactor.

### CODE REVIEW

**Review by sxa — COMMENTED** [review]
Does it need `.*` after `mac`?

**Comment by gdams** [comment]
> Does it need .* after mac?

@sxa not sure tbh, I just copied my code from the original PR which worked just fine

**Comment by gdams** [comment]
@sxa yup confirmed that works without the trailing `.*`

**Review by karianna — APPROVED** [review]
LGTM

File: jck/runtime.api/playlist.xml [hunk 1]
```diff
@@ -371,7 +371,7 @@
 			</disable>
  			<disable>
 				<comment>Requires UI https://github.com/temurin-compliance/temurin-compliance/issues/29</comment>
-				<platform>x86-64_mac</platform>
+				<platform>.*mac</platform>
```
Comment (smlambert) [inline jck/runtime.api/playlist.xml:374]:
```suggestion
				<platform>x86-64_mac|aarch64_mac</platform>
```

File: jck/runtime.api/playlist.xml [hunk 2]
```diff
@@ -416,7 +416,7 @@
 			</disable>
   			<disable>
 				<comment>Requires UI https://github.com/temurin-compliance/temurin-compliance/issues/29</comment>
-				<platform>x86-64_mac</platform>
+				<platform>.*mac</platform>
```
Comment (smlambert) [inline jck/runtime.api/playlist.xml:419]:
```suggestion
				<platform>x86-64_mac|aarch64_mac</platform>
```

File: jck/runtime.api/playlist.xml [hunk 3]
```diff
@@ -445,7 +445,7 @@
 			</disable>
 			<disable>
 				<comment>Requires UI https://github.com/temurin-compliance/temurin-compliance/issues/29</comment>
-				<platform>x86-64_mac</platform>
+				<platform>.*mac</platform>
```
Comment (smlambert) [inline jck/runtime.api/playlist.xml:448]:
```suggestion
				<platform>x86-64_mac|aarch64_mac</platform>
```

File: jck/runtime.api/playlist.xml [hunk 4]
```diff
@@ -512,7 +512,7 @@
 			</disable>
  			<disable>
 				<comment>Requires UI https://github.com/temurin-compliance/temurin-compliance/issues/29</comment>
-				<platform>x86-64_mac</platform>
+				<platform>.*mac</platform>
```
Comment (smlambert) [inline jck/runtime.api/playlist.xml:515]:
```suggestion
				<platform>x86-64_mac|aarch64_mac</platform>
```

File: jck/runtime.api/playlist.xml [hunk 5]
```diff
@@ -795,7 +795,7 @@
 			</disable>
 			<disable>
 				<comment>Requires UI https://github.com/temurin-compliance/temurin-compliance/issues/29</comment>
-				<platform>x86-64_mac</platform>
+				<platform>.*mac</platform>
```
Comment (smlambert) [inline jck/runtime.api/playlist.xml:798]:
```suggestion
				<platform>x86-64_mac|aarch64_mac</platform>
```

File: jck/runtime.api/playlist.xml [hunk 6]
```diff
@@ -942,7 +942,7 @@
...
```
Comment (smlambert) [inline jck/runtime.api/playlist.xml:945]:
```suggestion
				<platform>x86-64_mac|aarch64_mac</platform>
```

File: jck/runtime.api/playlist.xml [hunk 7]
```diff
@@ -1001,7 +1001,7 @@
...
```
Comment (smlambert) [inline jck/runtime.api/playlist.xml:1004]:
```suggestion
				<platform>x86-64_mac|aarch64_mac</platform>
```

File: jck/runtime.api/playlist.xml [hunk 8]
```diff
@@ -1084,7 +1084,7 @@
 			</disable>
  			<disable>
 				<comment>Requires UI https://github.com/temurin-compliance/temurin-compliance/issues/29</comment>
-				<platform>x86-64_mac</platform>
+				<platform>.*mac</platform>
```
Comment (smlambert) [inline jck/runtime.api/playlist.xml:1087]:
```suggestion
				<platform>x86-64_mac|aarch64_mac</platform>
```

File: jck/runtime.api/playlist.xml [hunk 9]
```diff
@@ -1127,7 +1127,7 @@
 			</disable>
 			<disable>
 				<comment>Requires UI https://github.com/temurin-compliance/temurin-compliance/issues/29</comment>
-				<platform>x86-64_mac</platform>
+				<platform>.*mac</platform>
```
Comment (smlambert) [inline jck/runtime.api/playlist.xml:1130]:
```suggestion
				<platform>x86-64_mac|aarch64_mac</platform>
```

**Review by smlambert — COMMENTED** [review]
I've made suggested changes, we shifted to prefer this style for clarity of reading the playlist exclusions.

**Review by smlambert — APPROVED** [review]
thanks @gdams

**Pull request merged**